### ISSUE TITLE
remnote plugin, here: Rollover Daily Todos

### ISSUE THREAD

**Issue by 22JoCo** [coldenate/rollover-daily-todos#20]
Dear remnote team, 

first of all: your programm is a game changer, and I am forever thankfull for you creating it!
Lately I tried out some plugins- including the todo- rollover (what a great idea! btw). But unfortunately my todos stay where they are. Am I doing sth wrong?

Thanks for your help in advance, 
Alle the best, 
Conny

**Comment by coldenate** [comment]
Hi! Thank you for stopping by! Could you please complete the following steps to give me an idea of what your KB looks like?

1. Does running the command `/rollover` manually do anything? Can we narrow this down to the Automatic part?
2. Can you send a screenshot or type out what settings you use in Rollover Daily Todos? 
3. What day are your Todos from? (Which daily document? can be multiple ofc)
4. Are you using Omnirollover at all?

Then, to try and fix the issue automatically:
1. Enable Debug Mode in the Plugin Settings
2. Run the command Debug Auto Rollover

**Comment by 22JoCo** [comment]
Dear coldnate!
Thanks for reaching out!
1. I tried to run the command manually- but it did not help, so, the problem consists in the manually as well as in the automatic mode
2. screenshot is underneath
3. the ToDos are mainly from the daily document
4. I installed Omnirollover a few days ago- since then it never worked. 

I did the set up like you wrote- and let you know if it worked out. 

Thanks so much for your help!
All the best, ￼
Constanze Koppe
